jmx_prometheus_javaagent/src/main/java/io/prometheus/jmx/JavaAgent.java

```diff
@@ -63,13 +63,19 @@ public static void premain(String agentArgument, Instrumentation instrumentation
                                     true,
                                     new File(config.file));
         } catch (ConfigurationException e) {
+            System.err.println(
+                    "Usage: -javaagent:/path/to/JavaAgent.jar=[host:]<port>:<yaml configuration file> ");
             System.err.println("Configuration Exception : " + e.getMessage());
             System.exit(1);
```

Comment:
Let's remove this catch code altogether since the underlying code could cause a `ConfigurationException` unrelated to usage (i.e. something wrong with the contents of the exporter YAML file.)